draft-quic-atsss-reqs.mkd

```diff
@@ -55,7 +55,7 @@ informative:
 
 --- abstract
 
-This document briefly presents the Access Traffic Steering, Switching, and Splitting (ATSSS) service being specified within the 3rd Generation Partnership Project (3GPP). The document also captures a set of data plane requirements to benefit from multipath capabilities. 
+This document briefly presents the Access Traffic Steering, Switching, and Splitting (ATSSS) service being specified within the 3rd Generation Partnership Project (3GPP). Future verssions of the document will capture a set of data plane requirements to benefit from multipath capabilities. 
```

Comment:
verssions -> versions besides that fine for me

Reply:
Cool. The nit was fixed here: https://github.com/obonaventure/draft-quic-atsss-reqs/pull/8/commits/fe473cd2409a2fd068f41be1a89c54981019eecc